### ISSUE TITLE
No font found, then crash

### ISSUE THREAD

**Issue by black-puppydog** [dskleingeld/pods#18]
Hi! I've been following along, seeing you be quite busy here!
Just wanted you to know you have a fan. :slightly_smiling_face: 

I just found the time to try and run the app on the pinephone (3GB version, running mobian with software at yesterday's updates) and I got this:

```
[Compile output]
     Running `target/debug/pods`
18:54:02 smithay_client_toolkit::window::concept_frame 658 ERROR: No font could be found
thread 'main' panicked at 'not yet implemented', src/page/podcasts.rs:46:21
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

This is on af3e1cf76131910eccc9b2f4a0f6632ef332ffd7.

From the user perspective it looked like this:

* the app opened as a blank page with only a textfield at the top (even though I thought I had successfully added a feed at my last attempt in Dec...? I may mis-remember this)
* I tried either pasting a url into the bar -> crash with above message
* Or typing the url by hand -> first few characters didn't crash it, but no text was shown either. But after `http://` or so it crashed with the same message.

The message makes me think I may be missing a font. Might just be a documentation issue? :crossed_fingers: 

Again, super happy you're taking this on. I can try and test a bit more from now on but as a dev I'm useless here. :P

**Comment by yara-blue** [comment]
Hey! 

Thanks a lot for trying it out. I am really happy to see you could build it! 

I hope to reach the first ([_minimal_app_](https://github.com/dskleingeld/pods/milestone/1)) this week, from then on it this should be usable as an ugly though responsive podcast app. I will do a release and write some documentation/usage instructions then, so you should get notified. In time I hope to build a nice looking intuitive UI that does not need instructions.

For now a nice looking UI is at the bottom of my list as I want to listen to podcasts again :grin: . From my own testing streaming episodes should work (only one though after one episode it buggs out) 

The error you got means I did not implement adding a podcast by Url yet. I do have a search option that "kinda" works, it needs an exact match with a part of the title of your podcast before it shows any results.

I do not have any time left today but if you check back later in the week it should be possible to add a podcast by typing in a feed Url.

**Comment by black-puppydog** [comment]
Ah okay, totally got that wrong then. I always assumed that adding by url is the trivial base case :D
Don't worry about ugly. I'm just happy that something is moving, and I love the energy you have for this! So keep it up, but don't burn out on it plz. :)

**Comment by yara-blue** [comment]
solved in #24 and #14
